src/gui_common/CustomRichTextLabel.tscn

```diff
@@ -11,8 +11,4 @@ theme_override_colors/default_color = Color(1, 1, 1, 1)
 theme_override_colors/font_shadow_color = Color(0, 0, 0, 0)
 theme_override_constants/line_separation = 5
 bbcode_enabled = true
-fit_content_height = true
```

Comment:
Seems like this property has been removed from Godot, which is why it was also removed from here. I hope that this means that nothing broke. To not break random text in the game I think it would not be very safe to enable "fit content" property here...